components/progress-indicator/private/step-vertical.jsx

```diff
@@ -0,0 +1,197 @@
+
+// Child component
+import { PROGRESS_INDICATOR_STEP_VERTICAL } from '../../../utilities/constants';
+import ButtonIcon from '../../icon/button-icon';
+
+// ### Display Name
+const displayName = PROGRESS_INDICATOR_STEP_VERTICAL;
+
+// ### Prop Types
+const propTypes = {
+	/**
+	 * **Assistive text for accessibility**
+	 * This object is merged with the default props object on every render.
+	 * * `completedStep`: Label for a completed step. The default is `Completed Step`
+	 * * `disabledStep`: Label for disabled step. The default is `Disabled Step`
+	 * * `errorStep`: Label for a step with an error. The default is `Error Step`
+	 * * `percentage`: Label for Progress Bar. The default is `Progress: [this.props.value]%`. You will need to calculate the percentage yourself if changing this string.
+	 * * `step`: Label for a step. It will be typically followed by the number of the step such as "Step 1".
+	 */
+	assistiveText: PropTypes.shape({
+		completedStep: PropTypes.string,
+		disabledStep: PropTypes.string,
+		percentage: PropTypes.string,
+		step: PropTypes.string,
+	}),
+	/**
+	 * Id for Steps, ranging in [0, steps.length).
+	 */
+	id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+	/**
+	 * Index of step. Used for id's if no step ID exists
+	 */
+	index: PropTypes.number,
+	/**
+	 * Determines if the step has been completed
+	 */
+
+	isCompleted: PropTypes.bool,
+	/**
+	 * Determines if the step has been disabled
+	 */
+	isDisabled: PropTypes.bool,
+	/**
+	 * Determines if the step contains an error
+	 */
+	isError: PropTypes.bool,
+	/**
+	 * Determines if the step is currently selected (active)
+	 */
+	isSelected: PropTypes.bool,
+	/**
+	 * Label of tooltip attached to the step if applicable.
+	 */
+	label: PropTypes.node,
+	/**
+	 * Triggered when click on individual steps. By default, it receives an event and returns all info passed to that step.
+	 * users are able to re-define it by passing a function as a prop
+	 */
+	onClick: PropTypes.func,
+	/**
+	 * Triggered when focus on individual steps. By default, it receives an event and returns all info passed to that step.
+	 * users are able to re-define it by passing a function as a prop
+	 */
+	onFocus: PropTypes.func,
+	/**
+	 * Determines the orientation of the progress indicator
+	 */
+	orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+	/**
+	 * Step object. This is passed into event callbacks.
+	 */
+	step: PropTypes.object,
+};
+
+/**
+ * StepVertical renders a button icon and its step label if applied
+ * The button is applied with different css classes under different conditions.
+ * Button icons have 4 types of status: completed (success), active (in progress), error (warning) and uncompleted (not approached)
+ */
+class StepVertical extends React.Component {
+	/**
+	 * buttonIcon represents the button icon used for each step.
+	 * the button is applied with different css classes under different conditions.
+	 */
+	buttonIcon(renderIcon, status, props) {
```

Comment:
Take another look at the SLDS markup. I believe these should be `span`s and not buttons since they are not tooltips anymore. https://www.lightningdesignsystem.com/components/progress-indicator/#Vertical

The vertical variant is not interactive at from what I can tell.